chore(website): Add case studies section (#10379)
cloudquery/cloudquery: website/pages/case-studies/add-your-case-study.mdx

---
title: "Add Your Case Study"

> date: 2032/06/07

Reviewer: ```suggestion
date: 2022/01/01
```

I think this should in fact be the last entry, not the first :) Otherwise it's a bit weird to land on the "case studies" page and the first thing you see is not a case study, but rather a request for case studies. Later we can make it a link that looks different from the rest, for now probably easiest to just keep it as the last entry.

Reviewer: Thinking about it again, I think it's also a bit strange that it says "Read the case study ->" under the li
![Screenshot 2023-04-27 at 15 35 59](https://user-images.githubusercontent.com/1121616/234896633-61401ada-c3c1-45dd-867f-beb8d7e14175.png)
nk

It would probably be best to keep this as a separate link outside of the index, so we don't have this

description: >-

>   In this post, Hexagon shares the details of how they built an Infrastructure Data Lake with CloudQuery

Reviewer: ```suggestion
  A template for adding a new case study to the CloudQuery website
```

author: yevgenypats
---

import { CaseStudyHeader } from "../../components/CaseStudyHeader"

<CaseStudyHeader/>

We would love to share your case study and use-case here!

> Feel free to use the below template and open a PR to this [repository](https://github.com/cloudquery/cloudquery/new/main/website/pages/how-to-guides)

Reviewer: ```suggestion
Feel free to use the below template and open a PR to this [repository](https://github.com/cloudquery/cloudquery/new/main/website/pages/case-studies)
```

> or if you want just share on the who's using [CloudQuery in Production Issue](https://github.com/cloudquery/cloudquery/issues/856)

Reviewer: ```suggestion
Alternatively, you can also share your use case in a less formal setting on the [Who's Using CloudQuery in Production](https://github.com/cloudquery/cloudquery/issues/856) section on GitHub.
```

> ## Introduction

Reviewer: Maybe we can put all this inside a code block?

> ## Introduction

Reviewer: ```suggestion
```markdown
## Introduction
```

> Share the back story of why you needed a tool like CloudQuery, what you were looking to achieve and

Reviewer: ```suggestion
Share the back-story of why you needed a tool like CloudQuery, what you were looking to achieve and
```

> what were some of the technical constraints.

Reviewer: ```suggestion
what some of the technical constraints were.
```

## Architecture

> Share the architecture of your deployment. Numbers, images, performance are always a great addition.

Reviewer: ```suggestion
Share the architecture of your deployment. Numbers, images and performance are always a great addition.
```

## Results

> What were some of the results you achieved and what some of the things you are planning to explore further.

Reviewer: ```suggestion
What were some of the results you achieved, and what are some of the things you are planning to explore further?
```

> 

Reviewer: ```suggestion
```
```
